scripts/scripts_object_storage/openstack_swift/local-setup-spark.sh

```diff
@@ -0,0 +1,10 @@
+ #!/usr/bin/env bash
+# Copyright 2016 Yahoo Inc.
+# Licensed under the terms of the Apache 2.0 license.
+# Please see LICENSE file in the project root for terms.
+#
+# This script install Spark locally
+
+wget http://archive.apache.org/dist/spark/spark-2.0.1/spark-2.0.1-bin-hadoop2.7.tgz
```

Comment:
Can use http://archive.apache.org/dist/spark/spark-2.0.0/spark-2.0.0-bin-hadoop2.7.tgz instead? It will reduce the changes.